scylla/src/transport/session.rs

```diff
@@ -29,66 +29,125 @@ pub struct Session {
     metrics: Arc<Metrics>,
 }
 
+/// Configuration options for [`Session`].  
+/// Can be created manually, but usually it's easier to use
+/// [SessionBuilder](super::session_builder::SessionBuilder)
 pub struct SessionConfig {
+    /// List of database servers known on Session startup.  
+    /// Session will connect to these nodes to retrieve information about other nodes in the cluster.  
+    /// Each node can be represented as a hostname or an IP address.  
     pub known_nodes: Vec<KnownNode>,
+
+    /// Preferred compression algorithm to use on connections.  
+    /// If it's not supported by database server Session will fall back to no compression.  
     pub compression: Option<Compression>,
-    // These configuration options will be added in the future:
-    // pub auth_username: Option<String>,
-    // pub auth_password: Option<String>,
+    /*
+    These configuration options will be added in the future:
+
+    pub auth_username: Option<String>,
+    pub auth_password: Option<String>,
 
-    // pub use_tls: bool,
-    // pub tls_certificate_path: Option<String>,
+    pub use_tls: bool,
+    pub tls_certificate_path: Option<String>,
 
-    // pub tcp_nodelay: bool,
-    // pub tcp_keepalive: bool,
+    pub tcp_nodelay: bool,
+    pub tcp_keepalive: bool,
 
-    // pub load_balancing: Option<String>,
-    // pub retry_policy: Option<String>,
+    pub load_balancing: Option<String>,
+    pub retry_policy: Option<String>,
 
-    // pub default_consistency: Option<String>,
+    pub default_consistency: Option<String>,
+    */
```

Comment:
Could we move the comment style change to the commit where those comments are introduced?